src/config.c

```diff
@@ -774,6 +776,7 @@ void configSetCommand(client *c) {
                         }
                     }
                     set_configs[i] = config;
+                    config_names[i] = config->name;
```

Comment:
maybe the string we give to the module should be a RedisModuleString, and we can use the one we got from argv so we don't need to copy any strings, but the module can still retain it if it wants?

Reply:
this I'm confused by.  I'm just reusing the string that was created for the config object (that also contains config alias, data to be set....).  Seems simpler than using the RedisModuleString

Reply:
yes, but maybe there's an advantage in passing a RedisModuleString.
although that could mean that the module will get a capital letters (whichever the user passed)